RCTVideo.m

```diff
@@ -238,12 +294,21 @@ - (void)setPaused:(BOOL)paused
     [self startProgressTimer];
     [_player play];
   }
-
+  
   _paused = paused;
 }
 
+
 - (void)setSeek:(float)seekTime
```

Comment:
Shouldn't `seek` be renamed to `currentTime`, see http://www.w3schools.com/tags/av_prop_currenttime.asp ?